swan/files/swan_config.py

```diff
@@ -26,8 +25,7 @@ def get_swan_user_pod(self):
 
         # pod labels
         pod_labels = dict(
-            lcg_release = self.spawner.user_options[self.spawner.lcg_rel_field].split('/')[0],
-            swan_user = self.spawner.user.name
```

Comment:
Can I ask one last thing? Separate this change into another commit, so you can explain that we remove this label because there is another label (you should say which one) that contains the user name.